packs/elastic/actions/show.snapshots.yaml

```diff
@@ -0,0 +1,82 @@
+---
+description: Show snapshots
+enabled: true
+entry_point: curator_runner.py
+name: show.snapshots
+parameters:
+  action:
+    default: show.snapshots
+    immutable: true
+    type: string
+  all_snapshots:
+    description: Do not filter snapshots.  Act on all snapshots.
+    type: boolean
+  dry_run:
+    default: false
+    description: Do not perform any changes.
+    type: boolean
+  exclude:
+    description: Snapshot list to operate on.
+    items:
+      type: string
+    type: array
+  host:
+    description: Elasticsearch host.
+    required: true
+    type: string
+  http_auth:
+    description: 'Use Basic Authentication ex: user:pass'
+    type: string
+  log_level:
```

Comment:
Maybe an `enum` type here then?

https://github.com/StackStorm/st2contrib/blob/b5c49e7b0dc2da0a9f29462c0e2adf80b443ea24/packs/jira/actions/create_issue.yaml#L15 for example.